src/plugins/CoreLoadPlugin.ts

```diff
@@ -70,7 +70,7 @@ export default class DojoLoadPlugin {
 		const bundleLoader = /bundle.*\!/;
 		const issuers: string[] = [];
 
-		compiler.apply(new NormalModuleReplacementPlugin(/dojo-core\/load\.js/, resolveMid('dojo-core/load/webpack')));
+		compiler.apply(new NormalModuleReplacementPlugin(/@dojo/core\/load\.js/, resolveMid('@dojo/core/load/webpack')));
```

Comment:
`@` is not a control/special character, so no, it should not need to be escaped.